docs/en/Getting-Started.md

```diff
@@ -252,14 +244,14 @@ dr-x------ staff  staff         4 PERSISTED 01-09-2018 16:34:55:362  DIR /mnt/s3
 -r-x------ staff  staff 157046046 PERSISTED 01-09-2018 16:35:01:002   0% /mnt/s3/sample_tweets_150m.csv
 ```
 
-This shows all the files under the root of the Alluxio file system from all of the mounted storage
+This shows all the files across all of the mounted storage
 systems. The `/LICENSE` file is from the local file system whereas the files under `/mnt/s3/` are
-from S3.
```

Comment:
uh this should be from still? from local file system vs from S3?? @aaudiber

Reply:
I think both should be "in"

The file metadata is in both Alluxio and S3, but the important part here is that Alluxio is letting us transparently view S3 files in the same namespace as files in the local filesystem. Alluxio isn't the owner of the files (data is 0% loaded), so the files aren't "from" S3/Local filesystem. They are still in S3/Local filesystem, Alluxio is just giving a view of them.